### DISCUSSION TITLE
is the twitter blog download still working?

### DISCUSSION THREAD

**Discussion by alexandralanorias** [TumblThreeApp/TumblThree#466] [Q&A (Question & Answer)]
hi hello good day, i would just like to ask whether or not the users can still download tweets from a blog in this app. the tumblr blog download feature works fine, but it's a different story altogether when twitter, or should i say x, is involved. the progress bar won't show any signs of life so i'm uncertain whether or not it's really scraping the tweets... feedback is much appreciated, thanks!

**Comment by thomas694** [comment]
Hello, I can still download blogs. As currently a request is only possible every approx. 12 seconds, it could look like it's not starting. Give it a moment, then it should show progress messages in the queue tab.
If not, you can have a look in the log file (%LOCALAPPDATA%\TumblThree\TumblThree.log) and see if there's an error message. You can also try to delete the `Cookies.json` in the settings folder and do a new login.

**Comment by Charlotte-br560** [comment]
Hello! The Twitter blog download feature may encounter issues due to changes in the platform's API or other factors. However, you can use *&lt;advertising&gt;*, a reliable web crawling tool, to extract tweets from Twitter blogs. It offers a user-friendly interface and efficient scraping capabilities. Give it a try and see if it resolves your issue. Feel free to let us know if you need any more help!
